modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -6,52 +6,15 @@
 
-* You enabled the required Technology Preview features for your cluster by editing the `FeatureGate` CR named `cluster`:
-+
-[source,terminal]
-----
-$ oc edit featuregate cluster
-----
-+
-.Example `FeatureGate` CR
-[source,yaml]
-----
-apiVersion: config.openshift.io/v1
-kind: FeatureGate
-metadata:
-  name: cluster
-spec:
-  featureSet: TechPreviewNoUpgrade <1>
-----
-<1> Enables the required `UserNamespacesSupport` and `ProcMountType` features.
-+
-[WARNING]
-====
-Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
-====
-+
-After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+For extra security, you can use the `restricted-v3` or `nested-container` SCC, which are specifically designed for workloads in Linux user namespaces. The `userNamespaceLevel: RequirePodLevel` field in the SCC requires that the workloads run in user namespaces. However, these SCCs do not use `RunAsAny` by default. For more information about SCCs, "Managing security context constraints".
 
-* The crun container runtime is present on the worker nodes. crun is currently the only OCI runtime packaged with {product-title} that supports user namespaces. crun is active by default.
-+
-[source,yaml]
-----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: ContainerRuntimeConfig
-metadata:
- name: enable-crun-worker
-spec:
- machineConfigPoolSelector:
-   matchLabels:
-     pools.operator.machineconfiguration.openshift.io/worker: "" <1>
- containerRuntimeConfig:
-   defaultRuntime: crun <2>
-----
-<1> Specifies the machine config pool label.
-<2> Specifies the container runtime to deploy.
+To require a specific SCC for a workload, set the `openshift.io/required-scc` annotation in the object specification. For more information, see "Configuring a workload to require a specific SCC". Alternatively, you can add an SCC to a specific user or group by using the `oc adm policy add-scc-to-user` or `oc adm policy add-scc-to-group` command. For more information, see the "OpenShift CLI administrator command reference".
+
+Also, you can optionally use the `procMount` parameter in a pod specification to configure the `/proc` file system in pods as `unmasked` within a Linux user namespace. Setting `/proc` to `unmasked`, which is generally considered as safe, bypasses the default masking behavior of the container runtime, and should be used only with an SCC that sets `hostUsers` to `false`. Running the `/proc` file system as `unmasked` within a user namespace has benefits that are beyond the scope of this documentation. 
```

Comment:
`within a Linux user namespace` isn't relevant. Technically speaking one could use Unmaked proc mount type outside of a userns, but we block it because it's giving the pod a lot of privilege